package/kernel/linux/modules/leds.mk

```diff
@@ -261,3 +261,37 @@ define KernelPackage/input-leds/description
 endef
 
 $(eval $(call KernelPackage,input-leds))
+
+
+define KernelPackage/leds-lp55xx-common
```

Comment:
This kmods needs to be a separate commit, see for example 77a14795dfd5488d62ec5f5cf04b7bd7a8c278ee or generally `git log package/kernel/linux/modules/leds.mk` for prior examples.